README.md

```diff
@@ -14,12 +14,114 @@ The shape framework (and shape canvas) allows to define relationship between var
 
 More over, the library source code is based on wxWidgets GUI toolkit, so it is platform independ (as far as the appropriate wxWidgets port is available for a required target platform).
 
+## Requirements:
+
+- wxSF uses wxWidgets (www.wxwidgets.org) and was tested with version 2.8.x on MS Windows and Linux (Ubuntu 7.04) target platforms.
+- Code::Blocks (www.codeblocks.org) IDE with MinGW C++ compiler or MS Visual Studio 2005/2008 Express are recommended for opening included workspace/project files but the library can be built from the command line as well.
+
 ## Distribution content:
 
 | folder | description |
 |---|---|
 |.\bin | Redistributable binary files needed for running of client applications using the wxSF's Dlls (contains basic wxWidgets library's dlls and C runtime libraries for both MINGW and MS VC++ 2008). |
-| .\demo | Standalone executable of wxShapeFramework demo app. |
-| .\doc | Library documentation. |
-| .\include | wxSF header files. |
-| .\lib | Prebuild wxSF libraries suitable for both static and dynamic linking and supported compilers (MinGW, MSVC8 and MSVC9) on MS Windows. |
+| .\doc | wxSF doxygen documentation file (in CHM and HTML format). |
+| .\include | wxSF headers files (needed for both hosting applications and library itself) |
+| .\lib | Lib files (will be created after library/demo app build) |
+| .\build | Premake build system for creation of various project/make files |
+| .\samples | Sample and demo projects with commented source code. |
+| .\src | Source code ad project files of the wxSF library |

```

Comment:
Thank you for the interest in making the repository better.  All changes looks good. But just nitpicking, there is a typo of the word "and" here